src/Compilers/Core/Portable/DocumentationCommentId.cs

```diff
@@ -599,23 +604,20 @@ public override bool VisitArrayType(IArrayTypeSymbol symbol)
                 }
 
                 _builder.Append("]");
-
-                return true;
             }
 
-            public override bool VisitPointerType(IPointerTypeSymbol symbol)
+            public override void VisitPointerType(IPointerTypeSymbol symbol)
             {
                 this.Visit(symbol.PointedAtType);
                 _builder.Append("*");
-                return true;
             }
 
-            public override bool VisitTypeParameter(ITypeParameterSymbol symbol)
+            public override void VisitTypeParameter(ITypeParameterSymbol symbol)
             {
                 if (!IsInScope(symbol))
                 {
                     // reference to type parameter not in scope, make explicit scope reference
-                    var declarer = new DeclarationGenerator(_builder);
+                    var declarer = new PrefixAndDeclarationGenerator(_builder);
```

Comment:
this is interesting.  we will generate something like `"T:Acme.OuterType`1:`0"`, even when generating a *reference*.  It feels wonky to me.  But we have tests for this.  So i guess it's intentional?  It's the only case for references though where we emit `prefix-colon` again...